### PR TITLE
wishlist-page-added

### PR DESCRIPTION
**Pull request by Jia-A**
Screenshot (459)
-- This is my wishlist page.
-- It has the cards/products which has been chosen by the user.
-- You can move these cards to cart or remove them from the wishlist.

### CODE REVIEW

**Review by FarhanMobashir — COMMENTED** [review]
code is clean. can be merged

**Review by FarhanMobashir — APPROVED** [review]

File: site-pages/wishlist/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+.footer{
+    text-align: center;
+}
+.wishlist-head{
+    font-weight: 500;
+}
+.wishlist-cards{
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
```
Comment (MaruthiPothuganti) [inline site-pages/wishlist/wishlist.css:16]: short-hand property can be used as _flex-flow: row wrap;_

**Review by MaruthiPothuganti — COMMENTED** [review]
Suggested a short-hand property. Everything else looks good.

File: site-pages/wishlist/wishlist.css (hunk 1 above)
Comment (Jia-A) [reply site-pages/wishlist/wishlist.css:16]: Okay, will change it...

**Pull request merged**